lib/browser/api/menu.js

```diff
@@ -125,18 +125,20 @@ Menu.prototype._init = function () {
       var checked, group, id, j, len, radioItem, ref1
       ref1 = this.groupsMap
       for (id in ref1) {
-        group = ref1[id]
-        checked = false
-        for (j = 0, len = group.length; j < len; j++) {
-          radioItem = group[j]
-          if (!radioItem.checked) {
-            continue
+        if(ref.hasOwnProperty(id))
```

Comment:
Actually, we should probably just use a `for (... of ...)` loop. What do you think?